modules/login/src/main/resources/META-INF/resources/login/login.jsp

```diff
@@ -1,199 +1,375 @@
+    <script src="login/js/bootstrap-select.min.js"></script>
+    <script src="login/js/validator.min.js"></script>
+    <script src="login/js/jquery.maskedinput.min.js"></script>
+    <script src="login/js/moment.min.js"></script>
+    <script src="login/js/registration.min.js"></script>
+    <script src="login/js/login.min.js"></script>
+</head>
+<body>
+
+<div class="container">
+    <form id="frmWorkspace" action="login" method="get">
+        <input type="hidden" id="currentWorkspace" name="currentWorkspace" />
+    </form>
+
+    <form id="login" action="login" method="post">
+        <div>
+            <div style="display: table;margin: 0 auto">
+                <div class="text-center"><img src="login/images/scout_logo.png" height="290px" /></div>
+                <div class="form-group form-inline text-center col-xs-12 hidden">
+                    <label for="method">Login Method:</label>
+                    <select id="method" name="method" class="form-control">
+                        <option value="">OpenAM</option>
                     </select>
                 </div>
-            </div>
-            
-            
-            <div class="content">
-                <div class="content-wrapper">
-                    <img src="login/images/scout_logo.png" height="290px" width="423px"></img>
-                    <br>
-                    <div class="field">
-                        <label for="method">Login Method:</label>
-                        <select id="method" name="method" tabindex="4">
-                            <option value="">OpenAM</option>
-                        </select>
-                    </div>
-                    <br>
-                    <div class="field">
-                        <label for="email">Email:</label>
-                        <br>
-                        <input type="text" id="email" name="email" tabindex="1" autofocus="autofocus"/>
-                    </div>
-                    <br>
-                    <div class="field">
-                        <label for="password">Password:</label>
-                        <br>
-                        <input type="password" id="password" name="password" tabindex="2" />
-                    </div>
-                    <br>
-                    <button type="submit" tabindex="3" >Login</button>
-                    <br/><br/>
-                    <span style="font-size: small">
-                    	Don't have an account? <a href="./register">Register</a>.
-                    </span>
-                    <br/><br/>
-                    <span style="font-size: small">
-                    	Forgotten Password? <a href="./forgotpassword">Click here</a>.
-                    </span>
+                <div id="workspaces" class="form-group text-center">
+                    <label for="server">Workspace:</label>
+                    <select id="server" class="form-control" name="workspace" style="width:auto;display: inline-block" required>
```

Comment:
Is there a reason many of the elements in this page have style attributes? Could those styles go in the stylesheet?